### ISSUE TITLE
vibe.d edit .dt files

### ISSUE THREAD

**Issue by EmTee70** [BBasile/Coedit#227]
Is there a way to edit the Diet template files ending with .dt.

(For your info: unfortunately a good support for editing vibe.d projects is missing in DlangIDE, too.)

**Comment by BBasile** [comment]
No. currently Coedit has two highlighters. One is for D (d di), the other is for everything else and it only highlights symbols and identifiers. Should i take this as a feature request ?

**Comment by EmTee70** [comment]
Yes, would be very cool, most important for .dt (vibe.d) is, that the type of indent is visible.
Because you have to avoid the mixing of tabs and spaces. It is very hard to repair a file with this type of error, if you don't see the tabs. 
When I opened my vibe project, the Coedit does not even show the .dt files in the directory view/ directory.

**Comment by EmTee70** [comment]
Just found the right options my self, for tab/space recognition:
Options=>Editor=> optios1=> eoShowSpecialChars True
and => eoTabsToSpaces = false.  
Now only some highlighting is missing :-)

**Comment by BBasile** [comment]
There is also now "options overrides" accessible from the editor context menu. One of them is to show the special chars, and then this doesn't affect the other editors.
